### PR TITLE
fixes 

### PR DESCRIPTION
**Pull request by carlosgonz0**

### CODE REVIEW

**Comment by carlosgonz0** [comment]
This still need fix, do you know how  delete previous  commits? I only want 
03add97

**Comment by elros34** [comment]
`git rebase -i HEAD~15` then replace "pick" with "drop" for commits which you want to delete

**Comment by carlosgonz0** [comment]
OoOOo  thank you.

**Comment by carlosgonz0** [comment]
Thank you @elros34, now i know how github work. Next time i will better on.

**Comment by carlosgonz0** [comment]
Seems still there are more issues, because my fork has  less commit that yours.

**Comment by carlosgonz0** [comment]
Looks is fixed now. Yes i do it. :  )

File: README.md [hunk 1]
```diff
@@ -1,31 +1,58 @@
...
-#### Want to contribute?
 
 - Review code!
 - Create issue for missing features!
 - Do something really cool!
 
-To build this app required:
 
-1. tdlibjson tdlibjson-devel [rpm](https://openrepos.net/content/blacksailer/tdlibjson) package installed 
-2. APP_ID and APP_HASH changed in tdlibjson_wrapper.pro
```
Comment (elros34) [inline README.md:29]: APP_ID and APP_HASH are required so please do not remove that step

File: README.md [hunk 2]
```diff
@@ -1,31 +1,58 @@
...
+    
+- Add the tdlibjson repo
+
+zypper ar http://repo.merproject.org/obs/home:/blacksailer:/branches:/home:/blacksailer/sailfish_latest_armv7hl/ tdlibsjon 
+    
+Refresh zypper repos
+
+zypper ref  
+
+- Install dependencies
```
Comment (elros34) [inline README.md]: All dependencies are installed automatically so this is not really needed but you can keep it if you want.

**Comment by elros34** [comment]
Maybe add WIP: to the title so I will know that PR is not ready to merge and remove it when you will finish your changes.

**Comment by carlosgonz0** [comment]
Ready to go.

**Comment by elros34** [comment]
OK I will merge it but some changes are incorrect so they will not work until i fix them.

**Comment by carlosgonz0** [comment]
Thank you @elros34.  Yeah there are some issues perhaps.  I will stop send PR without testing myself.

**Comment by carlosgonz0** [comment]
Wow pretty nice depecher v0.7.9. thank you elros34, you the best on depecher.

**Comment by carlosgonz0** [comment]
@elros34, seems that Pigg  prefered Depecher in your pinephone https://twitter.com/adampigg/status/1323392163095465984

**Pull request merged**